docs/runtime_options.md

```diff
@@ -134,7 +134,85 @@ cAdvisor stores the latest historical data in memory. How long of a history it s
 --storage_driver_user="root": database username (default "root")
 ```
 
-For storage driver specific instructions:
+## Perf Events
+
+```
+--perf_events_config="" Path to a JSON file containing configuration of perf events to measure. Empty value disables perf events measuring.
+```
+
+See example configuration below:
+```json
+{
+  "non_grouped": [
+    {
+      "type": 4,
+      "config": [
+        "0xc0"
+      ]
+    } 
+  ],
+  "grouped": [
+    [
+      {
+        "type": 13,
+        "config": [
+          "0xc04"
+        ]
+      },
+      {
+        "type": 13,
+        "config": [
+          "0xc304"
+        ]
+      }
+    ]
+  ],
+  "interval": "5s"
+}
+```
+
+In the example above number:
+* `INST_RETIRED.ANY_P` (number of instructions retired on 2nd Generation Intel® Xeon® Scalable Processor) identified by
```

Comment:
are events specific to processors?  Are there any common ones that nearly everyone would be able to use?

Reply:
Yes, events are different even between various generation of Xeons. If you take AMD's Naples and Rome CPUs you will see similar differences not mentioning other architectures such as ARM.

Reply:
I missed one of your questions: yes, some of the events (e.g. instructions retired, cycles) are supported across all or majority of Linux architectures.

Reply:
Great!  Once we have this in, it would be useful to have an example configmap in deploy/kubernetes as a follow-up, so it is easy for users to get started with the feature.